src/hooks/ProcessedRichNote/ProcessedRichNote.ts

```diff
@@ -22,43 +29,70 @@ export const useProcessedRichNote = (note: string | undefined) => {
    *   [www.wikipedia.org],
    * ]
    */
-  const linksPerLine = useMemo(() => {
-    return note?.split('\n').map(line => line.match(URLRegex)) ?? []
-  }, [note])
+  // Add newline before any links (starts with 'https' or 'ipfs') in the note that don't already have them
+  const noteParts = note?.split(/\s+/) ?? [] // split any whitespace (space or newline)
```

Comment:
Will this still allow multi line notes? or will it render something like:

```
Foo
Bar
```

as `Foo Bar`

Reply:
Correct. For now I've disabled new lines in the memo, another thing that's surely not necessary. If it's demanded we'll bring it back